MediaPlayer/MediaControls.js

```diff
@@ -185,7 +184,16 @@ const MediaControls = kind({
 		 * @default 'shuffle'
 		 * @public
 		 */
-		shuffleIcon: PropTypes.string
+		shuffleIcon: PropTypes.string,
+
+		/**
+		 * Specifies what kind of layout the MediaPlayer should have.
+		 *
+		 * @type {('full'|'tiny')}
+		 * @default 'full'
```

Comment:
I think we need to specify this value to `defaultProps` too. If it's used standalone, it will have no default value.

Reply:
Right. I can add it to defaultProps in mediaSlider and Times also